docs/source/releases.md

```diff
@@ -67,3 +67,30 @@ Support matrix table shows the version requirements for a particular **scylla-op
 | Scylla Manager    | `>=3.2.6`  | `>=3.2.6`  | `>=3.2`    | `>=2.6`    |
 | Scylla Monitoring | `(CRD)`    | `(CRD)`    | `(CRD)`    | `>=4.0`    |
 :::
+
+### Supported Kubernetes platforms
+
+We officially test and recommend to use the following platforms.
+
+:::{table}
+| Platform         | OS Image     |
+|:-----------------|:-------------|
+| GKE              | Ubuntu       |
+| EKS              | Amazon Linux |
+:::
+
+While our APIs generally work on any Kubernetes conformant cluster,
+performance tuning and other pieces that need to interact with the host OS, kubelet, cri, kernel, etc. might hit some incompatibilities.
+
+
+:::{warning}
+The following platforms are known **not to work correctly** at this time.
+This may change in the future.
```

Comment:
nit: imo that's redundant and also against [style recommendations ](https://docs.google.com/document/d/1lyHp1MKdyj0Hh3NprNFvEczA4dFSZIFoukGUvFJb9yE)

![image](https://github.com/scylladb/scylla-operator/assets/32443348/1b1c3373-2415-4204-895e-b1641eeebe86)

Reply:
removed